### PR TITLE
KNOP LED Custom setting and persistent setting store (BTT-TFT35-E3-V3.0)

### PR DESCRIPTION
**Pull request by GeminiServer**

### CODE REVIEW

**Comment by mehmetsutas** [comment]
What is KNOP led? Can you please provide some explanation?

**Comment by GeminiServer** [comment]
KNOP LED: BTT-TFT35-E3-V3.0 
Location Front: Rotationknob - RGB LED : D1,D2,D4,D5  - PIN 7

https://github.com/bigtreetech/BTT-TFT35-E3-V3.0/blob/master/Hardware/BTT%20TFT35-E3%20V3.0TOP.pdf

File: TFT/src/User/API/interfaceCmd.c [hunk 1]
```diff
@@ -310,7 +310,21 @@ void sendQueueCmd(void)
...
+            }
+            else
+            {
+              char buf[12];
+              sprintf(buf, "S%d\n", fanGetSpeed(1));
+              strcat(infoCmd.queue[infoCmd.index_r].gcode,(const char*)buf);
+              fanSetSendWaiting(1, false);
+            }
+            break;
+          }
```
Comment (guruathwal) [inline TFT/src/User/API/interfaceCmd.c]: does M710 exist in marlin firmware?

Comment (Scope666) [reply TFT/src/User/API/interfaceCmd.c]: Doesn't look like it:

https://marlinfw.org/meta/gcode/

Comment (GeminiServer) [reply TFT/src/User/API/interfaceCmd.c]: That's true: M710 is not in official marlin repo (yet). Sorry, accidentally commit that change. It is now removed from master. Thanks for the note.

**Pull request merged**